### ISSUE TITLE
Pagination state is not being saved when using yadcf

### ISSUE THREAD

**Issue by jds86930** [vedmack/yadcf#155]
I've been working on integrating this library in with one of my projects. It's working great, other than one thing. My tables (using DT 1.10.5) can have hundreds of rows & I just DT pagination (via DT's 'lengthMenu' option). When I integrated yadcf in, I noticed that GET/POST requests to the page that contains my table are causing pagination position to reset to page 1 (regardless of what page I was on). I know stateSave isn't being completely disregarded because sorting and page-length are being preserved; it seems like only pagination is being disregarded. This behavior was not present before integrating in yadcf, and it wasn't observed with the 'jquery-datatables-column-filter' either. I think the issue might be on display on your demos too (ex- http://yadcf-showcase.appspot.com/DOM_source.html). Feels like a bug in yadcf, but I could be missing something. Thoughts?

Example DT init:

...
        var alertSuspensionsTable = $('#AlertSuspensionsTable').DataTable({
            "lengthMenu": [[15, 30, 50, -1], [15, 30, 50, "All"]],
            "order": [[0, "asc"]],
            "autoWidth": false,
            "stateSave": true,
            "iCookieDuration": 2592000 // 30 days
        });
       
        yadcf_init_AlertSuspensionsTable(alertSuspensionsTable);

        var colvis = new $.fn.dataTable.ColVis(alertSuspensionsTable, {"align": "right", "iOverlayFade": 200});
        $(colvis.button()).prependTo('#AlertSuspensionsTable_filter');
...

```
function yadcf_init_AlertSuspensionsTable(alertSuspensionsTable) {
    yadcf.init(alertSuspensionsTable, [
        {column_number: 0, filter_reset_button_text: false, filter_type: "text", filter_default_label: "Filter"},
        {column_number: 1, filter_reset_button_text: false, filter_type: "select", data: ['Alert Name', 'Metric Group Tags', 'Everything'], sort_as: "none", filter_default_label: "Filter"},
        {column_number: 2, filter_reset_button_text: false, filter_type: "text", filter_default_label: "Filter"},
        {column_number: 3, filter_reset_button_text: false, filter_type: "text", filter_default_label: "Filter"},
        {column_number: 4, filter_reset_button_text: false, filter_type: "select", data: ['Yes', 'No'], sort_as: "none", filter_default_label: "Filter"},
        {column_number: 5, filter_reset_button_text: false, filter_type: "text", filter_default_label: "Filter"}
    ], 'footer');
}
```

**Comment by vedmack** [comment]
Fixed, grab the 0.8.7.beta.18 version of yadcf and the [nightly build of datatables](http://datatables.net/download/build/nightly/jquery.dataTables.min.js?_=331dc0e9b4f40281869dc42fe3728172) and you are set to go...

See in action : http://yadcf-showcase.appspot.com/DOM_Ajax_Multiple_1.10.html (first table)

**Comment by jds86930** [comment]
Thanks! I have verified that it is fixed for me in the 0.8.7.beta.18  w/ the 1.10.6 nightly of DT.

**Comment by sunde41** [comment]
Again, pagination state is not being saved when using yadcf.
I test DT 1.10.16...
